tests/DCPS/MetaStruct/MetaStructTest.cpp

```diff
@@ -75,6 +100,8 @@ int run_test(int, ACE_TCHAR*[])
   Source src;
   src.rhs_a.s = "hello";
   src.rhs_a.l = 42;
+  src.rhs_a.w = 0xdead;
```

Comment:
This will work, but I think it's fun to use non-ASCII when dealing with wide characters and strings. Me and @simpsont-oci have done this in the past. For example maybe U+263A (☺) or something else in [the Unicode Miscellaneous Symbols block](https://en.wikipedia.org/wiki/Miscellaneous_Symbols)?

```suggestion
  src.rhs_a.w = L'☺';
```